### DISCUSSION TITLE
do the drivers also work for the X52 H.O.T.A.S.

### DISCUSSION THREAD

**Discussion by WoutvanderAa** [nirenjan/x52pro-linux#53] [General]
Hey everyone, planning to buy the X52 H.O.T.A.S. but I am wondering if it also works for this hotas. thanks in advace!

**Comment by nirenjan** [comment]
If you are looking at this one https://a.co/d/bitmTTl, then yes, it should work just fine. 

For what it's worth, this program is not necessary in Linux, as the standard USB interface already supports the X52, but the mode switch is useless. For that, you need another program like antimicrox

The only driver functionality that libx52, as of version 0.3.2, supports are the mouse control via the thumbstick, brightness control, date and time on the MFD, and button color on the Pro. You can also write your own text on the MFD.

**Reply by WoutvanderAa** [reply]
> If you are looking at this one https://a.co/d/bitmTTl, then yes, it should work just fine.
> 
> For what it's worth, this program is not necessary in Linux, as the standard USB interface already supports the X52, but the mode switch is useless. For that, you need another program like antimicrox
> 
> The only driver functionality that libx52, as of version 0.3.2, supports are the mouse control via the thumbstick, brightness control, date and time on the MFD, and button color on the Pro. You can also write your own text on the MFD.

thanks for the information and the quick reply!
